java/network/cnp-egress-java-grails-tcp-protocol.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cilium.io/v2
+kind: CiliumNetworkPolicy
+metadata:
+  name: cnp-egress-java-grails-tcp-protocol
+  namespace: default # Change your namespace
+spec:
+  description: "Allow ingress communication on grails pod"
+  endpointSelector:
+    matchLabels:
+      name: grails # Change your matchLabels
+  egress:
+  - toPorts:
+    - ports:
+      - port: "443"
+        protocol: TCP
+    toCIDR:
```

Comment:
@tamilmaran-7 Not enforceable cc: @raviknox 
`error: error validating "https://raw.githubusercontent.com/kubearmor/policy-templates/2587bd0e0b10b8b543f818b124db7e3abafddc23/java/network/cnp-egress-java-grails-tcp-protocol.yaml": error validating data: ValidationError(CiliumNetworkPolicy.spec.egress[0].toCIDR[0]): invalid type for io.cilium.v2.CiliumNetworkPolicy.spec.egress.toCIDR: got "map", expected "string"; if you choose to ignore these errors, turn validation off with --validate=false`

Reply:
Check now @salman-accuknox cc @raviknox